### PR TITLE
[Docs] Update contributing guide

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/contributing/documentation/overview.rst [hunk 1]
```diff
@@ -71,7 +71,7 @@ To test the documentation before a commit:
 .. code-block:: bash
 
     $ pip install -r docs/requirements.txt
-    # This makes sure that the version of Sphinx you'll get is >=1.4.2!
+    # This makes sure that the version of Sphinx you'll get is >=1.6!
```
Comment (CoderMaggie) [inline docs/contributing/documentation/overview.rst]: Can we suggest installation via homebrew also?

Comment (Tomanhez) [reply docs/contributing/documentation/overview.rst]: I think that yes

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "docs/contributing-guide-documentation"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: docs/contributing/documentation/overview.rst [hunk 2]
```diff
@@ -71,13 +70,23 @@ To test the documentation before a commit:
...
+
+* Or by `Homebrew <https://brew.sh/>`_,
+
+.. code-block:: bash
+
+    $ brew install sphinx-doc
+
+.. tip::
+
+    Oficial guide installing Sphinx : `www.sphinx-doc.org <https://www.sphinx-doc.org/en/master/usage/installation.html>`_
```
Comment (CoderMaggie) [inline docs/contributing/documentation/overview.rst]:
```suggestion
    Oficial Sphinx installation guide : `www.sphinx-doc.org <https://www.sphinx-doc.org/en/master/usage/installation.html>`_
```

File: docs/contributing/documentation/overview.rst [hunk 3]
```diff
@@ -71,13 +70,23 @@ To test the documentation before a commit:
...
 
-* Install `Sphinx`_,
+* Install `Sphinx`_ by pip,
 
 .. code-block:: bash
 
-    $ pip install Sphinx
+    $ pip install -U sphinx
+
+* Or by `Homebrew <https://brew.sh/>`_,
```
Comment (CoderMaggie) [inline docs/contributing/documentation/overview.rst]: The choice of tool should be before line #66 of this file :)

**Comment by GSadee** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**